services/ui/ws/window_tree.cc

```diff
@@ -302,7 +302,7 @@ void WindowTree::OnWmMoveDragImageAck() {
 bool WindowTree::SetCapture(const ClientWindowId& client_window_id) {
   ServerWindow* window = GetWindowByClientId(client_window_id);
   WindowManagerDisplayRoot* display_root = GetWindowManagerDisplayRoot(window);
-  if (display_root && !display_root->window_manager_state())
+  if (display_root)
```

Comment:
For this and the one in WindowTree::ReleaseCapture there was no early return at all. I guess we can remove this display_root check unless to reduce the changeset. If that check turns out to be necessary, it should probably be upstream.